utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/wifi_stations.lua

```diff
@@ -3,10 +3,23 @@ local iwinfo = require "iwinfo"
 
 local function scrape()
   local metric_wifi_stations = metric("wifi_stations", "gauge")
+
   local metric_wifi_station_signal = metric("wifi_station_signal_dbm","gauge")
+
+  local metric_wifi_station_inactive = metric('wifi_station_inactive_ms', 'gauge')
+
+  local metric_wifi_station_exp_thr = metric('wifi_station_expected_throughput', 'gauge')
+
+  local metric_wifi_station_tx_bitrate = metric('wifi_station_tx_bitrate', 'gauge')
+  local metric_wifi_station_rx_bitrate = metric('wifi_station_rx_bitrate', 'gauge')
+
   local metric_wifi_station_tx_packets = metric("wifi_station_tx_packets_total","counter")
   local metric_wifi_station_rx_packets = metric("wifi_station_rx_packets_total","counter")
 
+  local metric_wifi_station_tx_bytes_total = metric('wifi_station_tx_bytes', 'counter')
+  local metric_wifi_station_rx_bytes_total = metric('wifi_station_rx_bytes', 'counter')
```

Comment:
`_total` needs to be added to the metric, not the variable name

`local metric_wifi_station_rx_bytes_total = metric('wifi_station_rx_bytes', 'counter')`
->
`local metric_wifi_station_rx_bytes = metric('wifi_station_rx_bytes_total', 'counter')`

Reply:
Oh. I see. No problem.